StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -473,27 +341,37 @@ public class BrushToolView: UIView {
     }
     
     @objc func selectColor(_ sender: UIButton) {
+        removeSubViews()
+        rootView.addSubview(groupSelectHexColorView)
+        
         setAllGroupdToolsButtonsAsNotClicked()
-        hideEraseSizeView()
-        hidePencilSizeView()
-        showSelectColorView()
         
         setButtonAsClicked(button: sender)
-        groupSelectHexColorView?.selectColorCollectionView.brushDelegate = delegate
     }
     
-    @objc func pencilSize(_ sender: UIButton) {
-        setButtonsAsNotClicked(buttons: groupPencilSizeButtons)
-        setButtonSizeAsClicked(button: sender)
-
-        delegate?.changePencilSize(CGFloat(sender.tag))
+    fileprivate func removeSubViews() {
+        for subView in rootView.subviews {
+            if subView.accessibilityIdentifier == GroupPencilSizeView.groupPencilSizeViewIdentifier {
```

Comment:
Does these if blocks can match in the same interaction? (_e.g._: a subview identifier can be GroupPencilSizeView.groupPencilSizeViewIdentifier and GroupEraseSizeView.groupEraseSizeViewIdentifier at the same time? If not, you should use if/else if blocks

Reply:
Done. Thanks! :D